packages/flame_svg/lib/svg_component.dart

```diff
@@ -7,18 +7,26 @@ import './svg.dart';
 /// Wraps [Svg] in a Flame component.
 class SvgComponent extends PositionComponent {
   /// The wrapped instance of [Svg].
-  Svg svg;
+  Svg? svg;
+
+  /// Creates an [SvgComponent]
+  SvgComponent({
+    this.svg,
+    Vector2? position,
+    Vector2? size,
+    int? priority,
+  }) : super(position: position, size: size, priority: priority);
 
   /// Creates an [SvgComponent] from an [Svg] instance.
   SvgComponent.fromSvg(
-    this.svg, {
+    Svg svg, {
```

Comment:
This constructor isn't needed anymore right?

Reply:
Right! but for the sake of avoid breaking changes I think we can keep it, I will mark it as deprecated, sounds good?